utils/webassembly/ci-linux.sh

```diff
@@ -34,8 +34,8 @@ sudo ./install_cmake.sh --skip-license --prefix=/opt/cmake
 sudo ln -sf /opt/cmake/bin/* /usr/local/bin
 cmake --version
 
-wget -O dist-wasi-sdk.tgz https://github.com/swiftwasm/wasi-sdk/suites/370986556/artifacts/809002
-unzip dist-wasi-sdk.tgz -d .
+wget -O dist-wasi-sdk.tgz.zip "https://github.com/swiftwasm/wasi-sdk/releases/download/0.1.0-swiftwasm/dist-ubuntu-latest.tgz.zip"
```

Comment:
Downloading Actions artifacts via direct link was invalidated, so I changed to use released artifacts